glide.go

```diff
@@ -719,12 +719,33 @@ Example:
 			},
 		},
 		{
-			Name:      "cache-clear",
-			ShortName: "cc",
-			Usage:     "Clears the Glide cache.",
-			Action: func(c *cli.Context) error {
-				action.CacheClear()
-				return nil
+			Name:  "cache",
+			Usage: "Manage Glide's cache",
+			Subcommands: []cli.Command{
+				{
+					Name:      "clear",
+					ShortName: "cc",
+					Usage:     "Clears the Glide cache",
+					Action: func(c *cli.Context) error {
+						action.CacheClear()
+						return nil
+					},
+				},
+				{
+					Name:  "path",
+					Usage: "Get a the path to a cached dependency's sources",
+					Description: `Returns the path to a cached dependency's
+		   sources. This is useful for scripts that need to run commands like
+		   "git-describe" on one of a project's vendored dependencies.
+
+		   For example,
+
+		       glide cache path github.com/emccode/libstorage`,
```

Comment:
Can you add a note that these are the dependency locations in the glide.yaml or glide.lock file? So that one knows what arguments should be passed in.